app/controllers/sessions_controller.rb

```diff
@@ -12,24 +10,48 @@ def destroy
     redirect_to root_url
   end
 
-  def googleAuth
-    # Manages the callback from Google
-    # Get access tokens from the google server
+  def omniauth_failure
+    redirect_to root_url
```

Comment:
should we perhaps flash a message at this point?

Reply:
Added!